device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/twin/MethodParser.java

```diff
@@ -267,17 +267,26 @@ else if (jsonElement instanceof JsonObject)
         }
     }
 
+    public JsonElement getPayloadFromJson(String json)
+    {
+        if (json == null || json.isEmpty())
+        {
+            return new JsonObject();
+        }
+        else
+        {
+            JsonElement jsonElement = new JsonParser().parse(json);
+            return jsonElement;
+        }
```

Comment:
Previous block returns. No `else` needed.
```suggestion

        JsonElement jsonElement = new JsonParser().parse(json);
        return jsonElement;
```